docker_mcp/server.py

```
@@ -8,11 +8,13 @@
 import argparse
 import os
 import sys
-from typing import Any
+from pathlib import Path
+from typing import Any, Annotated, Optional, List, Dict
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)